genesyscloud/resource_genesyscloud_outbound_sequence.go

```diff
@@ -120,6 +124,15 @@ func createOutboundSequence(ctx context.Context, d *schema.ResourceData, meta in
 
 	d.SetId(*outboundSequence.Id)
 
+	// Campaigns sequences can be enabled after creation
+	if status != "" && status == "on" {
```

Comment:
I guess this can just be `if status == "on"` ?

Reply:
Sure.